doc/source/data/inspecting-data.rst

```diff
@@ -143,3 +144,53 @@ of the returned batch, set ``batch_format``.
 For more information on working with batches, see
 :ref:`Transforming batches <transforming_batches>` and
 :ref:`Iterating over batches <iterating-over-batches>`.
+
+
+Inspecting runtime statistics
+=============================
+
+Ray Data calculates statistics during execution like runtime and memory usage for the different stages. 
```

Comment:
What is "runtime" in this context? Does it mean latency?

Reply:
updated to execution

Reply:
ok updated to wall clock time to make it more precise